### ISSUE TITLE
Multiple Display Issues

### ISSUE THREAD

**Issue by puppykickr** [TheRedSpy15/LTECleanerFOSS#77]
**Describe the bug**

There are multiple issues with the display that make the app difficult or even impossible to use.

**To Reproduce**

Opening the app on either of my very different devices will show that the display is defective in a few ways.

**Expected behavior**
Opening the app, the controls should be able to be seen.

When using certain features, the display should not cover up the results.

**Screenshots**

Will include screenshots if an update occurs and the problem persists.

**Smartphone (please complete the following information):**
Poblano VLE5 + ZTE 558VL
android 8.1.0 and 7.1.1
Lightning Browser
5.1.0

**Additional context**

I have already posted this issue under someone else's, but the OP was not very clear or specific with the issue.

Just in case, I went ahead and opened another issue and tried to make it easy to understand.

**Comment by TheRedSpy15** [comment]
The issue seems to be with 5 inch displays and smaller'

My testing 5.1 inch Galaxy s5 however works just fine strangely

If you could post images of the other screens, or at least the whitelist screen so I can better understand the issue. I don't have access to a 5 inch display myself

**Comment by puppykickr** [comment]
There are voth screens shown in the images.

The titles of the images lets you know which device the screen is on, and what the problem is.

Also please note that the Poblano has no dedicated navigation bar.

It pops up at the bottom of the screen, therefore completely covering up the 'clean' button.

On the ZTE, the circle denoting the percentage (which is overkill IMO, as the percentage is also shown numerically) is so large that it completely obsures the files that have been scanned.

**Comment by TheRedSpy15** [comment]
I believe I have finally fixed this!

Pushing an update to the beta branch on the play store tonight

**Comment by puppykickr** [comment]
@TheRedSpy15 

Did this issue get resolved?
There has been no update in F-Droid, and it appears that no update has appeared in the Play Store either.

**Comment by TheRedSpy15** [comment]
@puppykickr

Yeah, I meant to publish it in December but got really distracted with midterms. I committed the fix to github. It's just not published. I will make sure to do that today

**Comment by TheRedSpy15** [comment]
Rolling the update to the beta channel on the play store, and F-Droid will get around to it eventually on its own

**Comment by TheRedSpy15** [comment]
Can confirm it now works on 4.5 inch displays. Pushing to the stable channel on google play

**Comment by puppykickr** [comment]
Nice.
Thank you, and sorry for the aggrevation that this must have caused.
I will try it out when it becomes available.

Grant Ryan Swan
  (puppykicker)

**Comment by TheRedSpy15** [comment]
@puppykickr No aggrevation at all! I'm actually sorry for how long it took me to get around to finishing this 

The update is fully rolled out on both google play channels and on the github releases. Pending on fdroid

**Comment by puppykickr** [comment]
Just getting back to you about this.

It seems to be working just fine now.

In addition to the other devices mentioned above, I have revived an ancient (to me) ZTE Maven 812.

The app works just fine on it, as well.

Thanks for all the hard work on this!
